### ISSUE TITLE
DPL-1040 - As a developer (Ben) I would like traction to use the new sample sheet format for v13 revio runs. 

### ISSUE THREAD

**Issue by BenTopping** [sanger/traction-service#1178]
**User story**
As a developer (Ben) I would like traction to use the new sample sheet format for v13 revio runs.

**Who are the primary contacts for this story**
Ben T, Craig C

**Acceptance criteria**
To be considered successful the solution must allow:

- [ ] Replace v13_revio entry sample sheet in config/pipelines/pacbio.yml
  - Due to the new sample sheet structure you will need to edit the layout of this entry. Suggestion: split it into 3 csv_headesrs - [Run settings], [SMRT Cell Settings] and [Samples]. Each header will have subsequent fields and values. These fields work horizontally for Run settings and SMRT Cell settings and work vertically for samples. The horizontal fields will need new logic in sample sheet generation.
- See example sheet with proposed traction data. May require additional confirmation on exact data needed.
- [ ] - Update pacbio sample sheet generator or create a new one to handle new v13 structure.
- [ ] - Ensure v13_revio and v13_sequel_iie are still selectable in the UI and generates a sample sheet correctly.
- [ ] - Ensure the v13 sample sheets generated can be uploaded to SMRTLink

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/General-Backlog-Items/issues/358

**Comment by stevieing** [comment]
Page 28 documents the fields needed for a run https://www.pacb.com/wp-content/uploads/SMRT-Link-User-Guide-v13.0.pdf
